README.md

```diff
@@ -10,14 +10,207 @@ support new services is also a fairly simple process.
 
 ## Table of contents
 1. [Installation](#installation)
-2. [Documentation](http://docs.neuroinfo.org/lochness/en/latest/)
+2. [Quick setup from a template](#Setup from a template)
+3. [Documentation](http://docs.neuroinfo.org/lochness/en/latest/)
+
 
 ## Installation
-Just use `pip`
 
-```bash
-pip install lochness
+Just use ``pip`` ::
+
+    pip install lochness
+
+
+For most recent DPACC-lochness ::
+
+    pip install git+https://github.com/PREDICT-DPACC/lochness
+
+
+For debugging ::
+
+    cd ~
+    git clone https://github.com/PREDICT-DPACC/lochness
+    pip install -r ~/lochness/requirements.txt
+
+    export PATH=${PATH}:~/lochness/scripts  # add to ~/.bashrc
+    export PYTHONPATH=${PYTHONPATH}:~/lochness  # add to ~/.bashrc
+
+
+## Setup from a template
+
+### Creating the template
+Setting up lochness from scratch could be slightly confusing in the beginning.
+Try using the `lochness_create_template.py` to create a starting point.
+
+Create an example template to easily structure the lochness system
+
+```sh
+# PREDICT
+lochness_create_template.py \
+    --outdir /data/lochness_root \
+    --studies BWH McLean \
+    --sources redcap xnat box mindlamp \
+    --email kevincho@bwh.harvard.edu \
+    --poll_interval 43200 \
+    --ssh_host erisone.partners.org \
+    --ssh_user kc244
+
+# PRESCIENT
+lochness_create_template.py \
+    --outdir /data/lochness_root \
+    --studies BWH McLean \
+    --sources RPMS daris mediaflux mindlamp \
+    --email kevincho@bwh.harvard.edu \
+    --poll_interval 43200 \
+    --ssh_host erisone.partners.org \
+    --ssh_user kc244 
+
+# For more options: lochness_create_template.py -h
+```
+
+
+### Making edits to the template
+
+Running one of the commands above will create the structure below
+
+```sh
+/data/lochness_root/
+├── 1_encrypt_command.sh
```

Comment:
It is interesting that bash allows file names to start with a number but Python does not!